ingestors/cargo.go

```diff
@@ -36,7 +36,15 @@ func (ingestor *Cargo) Ingest() []data.PackageVersion {
 func (ingestor *Cargo) ingestURL(url string) []data.PackageVersion {
 	var results []data.PackageVersion
 
-	response, err := http.Get(url)
+    client := &http.Client{}
+    req, err := http.NewRequest("GET", url, nil)
```

Comment:
not-a-go-person question: if this is just setting up the request, not running it... how can it fail?